### PR TITLE
Mark ACTION, ACTIONG and ACTIONR as unsupported V2

### PR DESCRIPTION
**Pull request by gdfldm**
fodt-set-keyword-status --keyword ACTION --color orange 
fodt-set-keyword-status --keyword ACTIONG --color orange 
fodt-set-keyword-status --keyword ACTIONR --color orange

### CODE REVIEW

**Comment by gdfldm** [comment]
Replaces #221

File: parts/appendices/A.fodt [hunk 1]
```diff
@@ -6797,9 +6797,8 @@ Updated with AFR/TSA Rev-D comments and new keywords.</dc:description><meta:init
       <table:covered-table-cell/>
       <table:covered-table-cell/>
       <table:covered-table-cell/>
-      <table:table-cell table:style-name="Table35.H5" office:value-type="string">
-       <text:p text:style-name="P2480"/>
-      </table:table-cell>
+      <table:table-cell table:style-name="Table38.H21" office:value-type="string">
+       <text:p text:style-name="P2480"/>      </table:table-cell>
```
Comment (blattms) [inline parts/appendices/A.fodt:6801]: Strange, it seems like the script removes a  line break, @hakonhagland

Comment (hakonhagland) [reply parts/appendices/A.fodt:6801]: @blattms Yes that seems strange.. Let me check what is going on.

Comment (hakonhagland) [reply parts/appendices/A.fodt:6801]: Think I found the issue, see #248.

**Pull request merged**